Epsilon.Abstractions/LearningDomainOutcomeResult.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Epsilon.Abstractions;
 
-public record LearningDomainOutcomeRecord(
+public record LearningDomainOutcomeResult(
```

Comment:
Just want to mention. That the name was moved away from `LearningDomainOutcomeResult` because this record will also be used if an outcome has not been graded yet. Thus, not being a result at that point.

Reply:
Ah yea, forgot about that one. It wasn't updated in the codebase yet because in some places it was still called result, so I wanted to correct the inconsistent naming.